static/app/components/onboarding/gettingStartedDoc/types.ts

```diff
@@ -87,6 +87,7 @@ export interface Docs<PlatformOptions extends BasePlatformOptions = BasePlatform
   feedbackOnboardingNpm?: OnboardingConfig<PlatformOptions>;
   platformOptions?: PlatformOptions;
   replayOnboardingJsLoader?: OnboardingConfig<PlatformOptions>;
+  replayOnboardingMobile?: OnboardingConfig<PlatformOptions>;
   replayOnboardingNpm?: OnboardingConfig<PlatformOptions>;
```

Comment:
Is there a reason why the mobile docs need a separate key?
Couldn't we use `replayOnboarding` as default key and additionally set `replayOnboardingJsLoader` if the platform supports the loader script? 🤔